### PR TITLE
Feature/error reporting

### PR DESCRIPTION
**Pull request by Maurice22**
**What changes does this PR introduce**

Adds error reporting as discussed in #158 .

Adds sentry and links it to the QH project. Added manual error reporting at places we currently catch errors already.

**Checklist**
- [ ] `yarn build` passes
- [ ] `yarn lint` does not show any errors

**Others details**
- [x] This PR introduces a new dependency: @sentry/browser required for the sentry error reporting

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -42,7 +43,11 @@ function App(props) {
   } = props;
 
 
-  const addListener = () => {
+  const enableAnalytics = () => {
+    // Crash reportin
+    Sentry.init({ dsn: 'https://d00e781dba684e6e93070bcb8f2da4c6@sentry.io/5178489' });
```
Comment (florianschmidt1994) [inline src/App.jsx]: Can we use .env files for this?

File: src/views/NotifyMe.jsx [hunk 2]
```diff
@@ -29,6 +30,7 @@ export default function NotifyMe() {
 
       setSignInLinkSent(true);
     } catch (error) {
+      Sentry.captureException(error);
```
Comment (florianschmidt1994) [inline src/views/NotifyMe.jsx]: What happens here if the cookie consent has not been accepted?

Comment (Maurice22) [reply src/views/NotifyMe.jsx]: No errors are shown and the errors are also not written into a backlog. So it's basically a noop.

File: src/App.jsx [hunk 3]
```diff
@@ -42,7 +43,11 @@ function App(props) {
   } = props;
 
 
-  const addListener = () => {
+  const enableAnalytics = () => {
+    // Crash reportin
```
Comment (jagaSto) [inline src/App.jsx]: typo, should be 
```suggestion
    // Crash reporting
```

:)

**Comment by Maurice22** [comment]
Good point, switched to environments instead of projecs

**Comment by florianschmidt1994** [comment]
Looks good, will merge!

**Pull request merged**